accounts/models.py

```
@@ -39,5 +54,94 @@ class Account(models.Model):
+
+    async def fetch_near_social_profile_data_async(self):
+        fetch_profile_data = sync_to_async(self.fetch_near_social_profile_data)
+        await fetch_profile_data()
+
+    def fetch_near_social_profile_data(self, should_save=True):
+        # Fetch social profile data from NEAR blockchain
+        try:
+            url = f"{settings.FASTNEAR_RPC_URL}/account/{settings.NEAR_SOCIAL_CONTRACT_ADDRESS}/view/get"
+            keys_value = f'["{self.id}/profile/**"]'
+            params = {"keys.json": keys_value}
+            response = requests.get(url, params=params)
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)